### PR TITLE
DM-35047: Add upload to pypi

### PR DESCRIPTION
**Pull request by mwittgen**
## Checklist

- [ ] ran Jenkins
- [ ] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,18 +1,25 @@
 [metadata]
-name = ctrl_mpexec
+name = lsst-ctrl-mpexec
 description = Pipeline execution infrastructure for the Rubin Science Pipelines
 author = Rubin Observatory Data Management
 url = https://github.com/lsst/resources
```
Comment (timj) [inline setup.cfg:5]: Wrong URL

File: setup.cfg [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+author_email = dm-admin@lists.lsst.org
+license = GPLv3+ License
 classifiers =
     Intended Audience :: Science/Research
-    License :: OSI Approved ::  GNU General Public License v3 or later (GPLv3+)
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
     Operating System :: OS Independent
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.8
```
Comment (timj) [inline setup.cfg:13]: I thought this now failed on python 3.8?

**Review by timj — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (mwittgen) [reply setup.cfg:5]: So this should use solely python 3.10?

Comment (timj) [reply setup.cfg:5]: I think you are replying to the other comment...

File: setup.cfg (hunk 2 above)
Comment (mwittgen) [reply setup.cfg:13]: So this should use 3.10?

Comment (timj) [reply setup.cfg:13]: I thought you were testing the different python versions before declaring them to work. I think ctrl_mpexec has a problem with 3.8 type annotations (found by ctrl_bps). Maybe declare 3.10 then but in a follow up ticket we should enable 3.8, 9, 10 matrix and allow us to properly state which versions we support (we are not actively preventing 3.8 from working but if it stops working we do not care).

Comment (mwittgen) [reply setup.cfg:13]: OK.

**Pull request merged**